Bmx.Core/BmxCore.cs

```diff
@@ -70,7 +62,25 @@ public class BmxCore {
 				// TODO: Remove need for input here, Push style flows have no user input on app here
 				// ex: https://developer.okta.com/docs/reference/api/authn/#response-example-waiting
 				PromptMfaInput( selectedMfa.Name );
+				throw new NotImplementedException();
 			}
+
+
+			// TODO: Decouple this more, there is an implicit understanding its an Okta AWS app
+			var accounts = await _identityProvider.GetAccounts( "amazon_aws" );
+			account = account?.ToLower();
+
+			int selectedAccountIndex = -1;
+
+			if( account != null ) {
+				selectedAccountIndex = Array.IndexOf( accounts.Select( s => s.ToLower() ).ToArray(), account );
+			}
+
+			if( account == null || selectedAccountIndex == -1 ) {
+				selectedAccountIndex = PromptAccountSelection( accounts );
+			}
```

Comment:
Not sure if you were planning to do this in another PR, but thoughts on logging a message if an account isn't found?

Reply:
Yep, there is a lot more to be done in terms of handling things that aren't on the happy path:
- Handling API errors from Okta (due to invalid values passed for credentials, mfa tokens etc...) or this
- Handling any other type of error, unable to contact Okta or expired sessions.

Thinking of handling them by having the Identity or Service providers throw exceptions that Bmx Core can notify the app (in this case CommandLineApp to notify the user about before exiting).

.NET Core has good handling for easily swappable logging, so an actual log to file might be a good idea as well. Will create an issue for this.